cosmic/tests/data/Params.ini

```diff
@@ -13,14 +9,55 @@ select_final_state = True
 ; default=[0,1,2]
 binary_state = [0,1,2]
 
-; lisa_sources decides whether to retain only sources within LISA's frequency range (porb<1e5 sec)
-; default=False
-lisa_sources = False
+[sampling]
+; Specify if you woud like to sample initial conditions via
+; the independent method (indepedent) or would like to sample
+; initial conditions follow Moe & Di Stefano (2017) (multidim)
+initial_samp=multidim
+
+; Galaxy Components. Options include Bulge, ThinDisk, ThickDisk, and DeltaBurst
+; Think Star Formation History with which to use as your basis for sampling
+galaxy_component = DeltaBurst
+
+; Metallicity of the population of initial binaries
+metallicity = 0.02
 
 [convergence]
-; perform convergence over porb < 5000 sec for LISA Galaxies
-; default=False
-lisa_convergence=False
+; A list of parameters you would like to verify have converged
+; to a single distribution shape.
+; Options include mass_1, mass_2, sep, porb, ecc, massc_1, massc_2
+; rad_1, rad_2
+convergence_params = [mass_1,mass_2,porb,ecc]
+
+; formation computes the convergence on the formation properties
+; of the binaries with user specified final kstars if True
+
+; 1_SN computes the convergence on the binary properties 
+; just before the first super nova on the population with 
+; user specified final kstars if True
+
+; 2_SN computes the convergence on the binary properties 
+; just before the second super nova on the population with 
+; user specified final kstars if True
```

Comment:
; 2_SN computes convergence on binary properties
; just before the second supernova for the population with
; user-specified final kstars if True